providers/edge/src/airflow/providers/edge/models/edge_worker.py

```diff
@@ -184,11 +187,12 @@ def reset_metrics(worker_name: str) -> None:
 
 
 @provide_session
-def request_maintenance(worker_name: str, session: Session = NEW_SESSION) -> None:
+def request_maintenance(worker_name: str, maintenance_comment: str, session: Session = NEW_SESSION) -> None:
```

Comment:
Optional here as well
```suggestion
def request_maintenance(worker_name: str, maintenance_comment: str | None, session: Session = NEW_SESSION) -> None:
```